cristin-import/src/main/java/no/unit/nva/cristin/mapper/PeriodicalBuilder.java

```diff
@@ -34,11 +32,9 @@ private Periodical createUnconfirmedJournal() {
     }
 
     private Periodical createJournal() {
-        URI journalUri = new UriWrapper(NVA_API_DOMAIN).addChild(MappingConstants.NSD_PROXY_PATH)
-            .addChild(MappingConstants.NSD_PROXY_PATH_JOURNAL)
-            .addChild(cristinObject.getJournalPublication().getJournal().getNsdCode().toString())
-            .addChild(cristinObject.getPublicationYear().toString())
-            .getUri();
+        Integer nsdCode = cristinObject.getJournalPublication().getJournal().getNsdCode();
+        int publicationYear = cristinObject.getPublicationYear();
```

Comment:
Maybe just me, but is this the correct target — from the conversation with Kari and Daniel, I'd assumed this should be "reporting year", but I may have misunderstood

Reply:
reporting year is not always present. I can try to fix this.